src/db/migrations/20250219182508_add_notifications_table.js

```diff
@@ -0,0 +1,32 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+export function up(knex) {
+  return knex.schema.createTable("notifications", (table) => {
+    table.increments("notification_id").primary();
+    table.string("label").nullable();
+    table.string("title").nullable();
+    table.string("description").nullable();
+    table.string("small_image_path").nullable();
+    table.string("big_image_path").nullable();
+    table.string("cta_label").nullable();
+    table.string("cta_link").nullable();
+    table.string("string").nullable();
```

Comment:
Is this an actual column that's needed? :sweat_smile:

Reply:
I'm assuming this wasn't a reply to my question about having a column called `string`, but to my question about the type definitions.

So just for context, the reason for that is that we can then roll back the unrelated changes without having to remove the migrations (because Knex complains if there are migrations logged in the DB for which there are no files in the repo). Since the type definitions are related to the migrations (i.e. they define what the DB look like after the migrations), it's fine to include them with the migration PR/commit.